pkg/httpd/websftp.go

```diff
@@ -91,6 +141,9 @@ func (h *webSftp) dispatch(msg *Message) {
 		message.Err = "Unknown command"
 		h.ws.SendMessage(&message)
 	}
+
+	h.isConnected = true
+
 }
 
 func (h *webSftp) handleList(request *webSftpRequest, response *Message) {
```

Comment:
There seems to be an issue with the code provided here; it includes commented-out Go language comments in place of actual logic, some variable names are inconsistent or incorrect, indentation is off, and there isn't a clear explanation of what this file represents. It would be helpful if you could provide more context so I can assist properly.

I don't see any obvious issues but please make sure there's no intentional confusion due to missing or incorrectly written code sections present in the original example.